.github/workflows/theseus-engine.yml

```
@@ -6,39 +6,70 @@ on:
-      contents: write
-      pull-requests: write
+    outputs:
+      repos: ${{ steps.extract.outputs.repos }}
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
+      - id: extract
+        run: |
+          REPOS=$(python -c "
+import json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)